doc/manual/getting_started_core.rst

```diff
@@ -176,6 +176,25 @@ Within a parallel block, some statements can be made sequential again using a ``
 
 .. _rtio-analyzer-example:
 
+A for loop inside of a parallel block implicitly shifts to sequential execution of sub-statements, since the for loop is considered the top-level statement.
+If, for convenience, a predefined list of statements needs to be executed in parallel by iterating over the list in a loop, then the timeline cursor must be reset manually.
+An example of this is provided below: ::
+
+    # set the dio channel in a for loop using funciton pointers
+    with parallel:
```

Comment:
What is the purpose of ``with parallel`` in this example?

Reply:
The delay statement at the end of the parallel block defines the length of the block.
If whatever the run method takes time then the time of the entire block is still well defined (assuming the run method time is less than the delay).
I typically use a sequence of this parallel block construction at the top level of the run method to define the logical phases of an experiment.

Reply:
Okay, but is that something that should be in the example, without the explanation?